parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -112,9 +112,18 @@ configureSwapFile() {
     SWAP_SIZE_KB=$(expr {{GetSwapFileSizeMB}} \* 1000)
     DISK_FREE_KB=$(df /dev/sda1 | sed 1d | awk '{print $4}')
     if [[ ${DISK_FREE_KB} -gt ${SWAP_SIZE_KB} ]]; then
-        retrycmd_if_failure 24 5 25 fallocate -l ${SWAP_SIZE_KB}M /swapfile
+        function setupSwap() {
+            fallocate -l ${SWAP_SIZE_KB}M /swapfile
+            chmod 600 /swapfile
+            mkswap /swapfile
+            swapon /swapfile
+        }
+        export -f setupSwap
+        retrycmd_if_failure 24 5 25 bash -c setupSwap || exit $ERR_SWAP_CREAT_FAIL
```

Comment:
can we do a check after the setup
sudo swapon --show

Reply:
added swapon --show to check